### PR TITLE
Fix Issue #145: Change spacer_between_sections type to string...

### PR DESCRIPTION
**Pull request by mttmzr**
and update`lib/docblockr-worker.js` accordingly.

### CODE REVIEW

**Comment by felixkempf** [comment]
+1

**Comment by cringer** [comment]
+1

**Comment by curtisblackwell** [comment]
:+1:

**Comment by soderlind** [comment]
+1

**Comment by garygreen** [comment]
+1. Looks like this may fix #208

File: README.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ You can access the configuration settings by entering `Docblockr` in atom settin
 
 - `return_tag` *(String)* The text which should be used for a `@return` tag. By default, `@return` is used, however this can be changed to `@returns` if you use that style.
 
-- `spacer_between_sections` *(Boolean|String)* If true, then extra blank lines are inserted between the sections of the docblock. If set to `"after_description"` then a spacer will only be added between the description and the first tag. Default: `false`.
+- `spacer_between_sections` *(String)* If true, then extra blank lines are inserted between the sections of the docblock. If set to `"after_description"` then a spacer will only be added between the description and the first tag. Possible values are `"yes"`, `"no"` and `"after_description"`. Default: `"no"`.
```
Comment (albertorestifo) [inline README.md]: > If **set to yes**, then …

**Comment by albertorestifo** [comment]
@garygreen The problem is that when setting a paramenter as a boolean, Atom displays a checkbox so you have no way to set it to any other value.

Yes, this would be a breaking changes but given that noone was able to set it to `after_description` I don't think it's a big problem.

What about adding a separate boolean configuration setting, "Insert space after decsription"?

**Comment by thinkmattm** [comment]
Sorry for the (very) late response.

@albertorestifo is right. If I remember correctly, I tried to set the config setting to `type: ['boolean', 'string']` and `enum: [true, false, 'after_description']`, which should theoretically work, but Atom didn't seem to like that. The generated select menu did not show the boolean values at all. Maybe it's a bug in Atom itself?

But @garygreen is right, too. So let's just use `true` and `false` as strings instead of `yes` and `no`?

**Comment by arm22** [comment]
+1

**Comment by MoritzKn** [comment]
Sorry for the (very very) late response. 
This seams reasonable, thanks for your contribution!

**Pull request merged**